### PR TITLE
New car details

### PR DESCRIPTION
**Pull request by koalalorenzo**
I've added some more car details, including the last update, the car type, mileage and the car purpose (Private, transport, etc etc). Improved also readability of the code by reducing the time we are catching errors in the code (but using a method that is getting the text value out of an element of the page)

### CODE REVIEW

**Comment by koalalorenzo** [comment]
I will update tests soon.

File: dk_car_scraper/scraper.py [hunk 1]
```diff
@@ -11,12 +10,27 @@
...
+    """Returns the value inside the content"""
+    try:
+        value = content
+        for index in list(args):
+            value = value[index]
+
+        if value != content and type(value) == element.Tag:
+            return True, value.text
+
+    except (IndexError, KeyError, AttributeError):
```
Comment (joshuakarjala) [inline dk_car_scraper/scraper.py:23]: if exception why not just return False?
and otherwise return value

Comment (koalalorenzo) [inline dk_car_scraper/scraper.py:23]: Is started thinking that this method could return any type of value (including False, None and that value). Then I changed it to text only, but now it is useful to understand if the exception was correct or not, even if the value is none.

**Comment by koalalorenzo** [comment]
@joshuakarjala Are you going to test this and release it with a new version? :+1:

**Comment by joshuakarjala** [comment]
I presume you tested the code that you put in ;)

**Comment by koalalorenzo** [comment]
Yes I did :-) I also added few cars/license in the tests to be sure that some other "kind" of car are working properly.

**Pull request merged**